package.json

```diff
@@ -1,21 +1,34 @@
 {
+  "name": "react-final-table",
   "version": "1.4.2",
   "license": "MIT",
   "author": "Gabriel Abud",
+  "keywords": [
+    "react",
+    "hooks",
+    "typescript",
+    "table",
+    "headless",
+    "react-hooks"
+  ],
   "main": "dist/index.js",
-  "typings": "dist/index.d.ts",
+  "module": "dist/react-final-table.esm.js",
+  "types": "dist/index.d.ts",
   "files": [
     "dist",
     "src"
   ],
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/react-final-table.esm.js"
+    }
+  },
+  "sideEffects": false,
```

Comment:
This uses TSDX, which uses Rollup not Webpack from my understanding.  It looks like treeshaking is already enabled by default: https://tsdx.io/

Reply:
Correct, for this library. It does some tree-shaking.

> AS you can see, TSDX stripped out the development check from the production code. This allows you to safely add development-only behavior (like more useful error messages) without any production bundle size impact.

> For ESM build, it's up to end-user to build environment specific build with NODE_ENV replace (done by Webpack 4 automatically).

However, this won't/can't tree-shake exports since it won't know what is unused until the developer uses the library. So if the developer uses Webpack this library won't tree-shake properly without this flag.